image/cli/mascli/functions/provision_fyre

```diff
@@ -440,6 +440,11 @@ function provision_fyre() {
     export OCP_CONSOLE_URL="https://$OCP_CONSOLE_ROUTE"
     rm /tmp/fyre-$CLUSTER_NAME.json
   fi
-  mas-devops-notify-slack --action ocp-provision-fyre --rc $result
+  if [[ -n $MSG ]]; then
```

Comment:
`MSG` is very generic, risk of overload/conflict with other env vars ... if the message is being passed as an env var it should be something more specific like `NOTIFY_SLACK_MSG` that makes it obvious the purpose.